### PR TITLE
React/dp 17340 add typography

### PR DESCRIPTION
**Pull request by clairesunstudio**
Added:
  - project: React
    component: Typography
    description: Added Typography story and documentation. (#918)
    issue: DP-17340
    impact: Patch
  - project: React, Patternlab
    component: styles
    description: Added `ma-texta-style($weight, $italic)` mixin and Texta font style classes. (#918)
    issue: DP-17340
    impact: Patch
  - project: React, Patternlab
    component: styles
    description: Added `$fonts` and `$fonts-mono` font family variables. (#918)
    issue: DP-17340
    impact: Patch

----

Screen Shot 2020-02-04 at 12 46 20 PM
Screen Shot 2020-02-04 at 12 46 26 PM
Screen Shot 2020-02-04 at 12 46 31 PM

### CODE REVIEW

File: changelogs/DP-17340.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Added:
+  - project: React
+    component: Storybook
```
Comment (clairesunstudio) [inline changelogs/DP-17340.yml]:
```suggestion
    component: Typography
```

File: react/.storybook/webpack.config.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ module.exports = ({ config }) => {
     alias: {
       ...config.resolve.alias,
       SharedAssets: path.resolve(__dirname, '../../assets/'),
+      GitbookAssets: path.resolve(__dirname, '../../docs/.gitbook/assets'),
```
Comment (rbayliss) [inline react/.storybook/webpack.config.js]: Is this still needed?

File: react/src/.gitbook/assets [hunk 3]
```diff
@@ -0,0 +1 @@
+../../../docs/.gitbook/assets
```
Comment (rbayliss) [inline react/src/.gitbook/assets]: Is this still needed?

**Review by rbayliss — COMMENTED** [review]
Hey @clairesunstudio, this looks good - I think there are a couple of leftover things that should be cleaned up though.

File: react/.storybook/webpack.config.js (hunk 2 above)
Comment (clairesunstudio) [reply react/.storybook/webpack.config.js]: nope, removed in d0c5aa7

File: react/src/.gitbook/assets (hunk 3 above)
Comment (clairesunstudio) [reply react/src/.gitbook/assets]: removed in d0c5aa7

**Review by rbayliss — APPROVED** [review]

**Pull request merged**